client/signup/steps/woocommerce-install/select-site/index.tsx

```diff
@@ -0,0 +1,64 @@
+import { isBusinessPlan } from '@automattic/calypso-products';
+import { Card } from '@automattic/components';
+import { useI18n } from '@wordpress/react-i18n';
+import { ReactElement, useState, useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import SiteSelector from 'calypso/components/site-selector';
+import StepWrapper from 'calypso/signup/step-wrapper';
+import { submitSignupStep } from 'calypso/state/signup/progress/actions';
+import type { WooCommerceInstallProps } from '../';
+
+import './style.scss';
+
+interface Site {
+	capabilities: {
+		manage_options: boolean;
+	};
+	jetpack: boolean;
+	plan: {
+		product_slug: string;
+	};
+}
+
+export default function SelectSite( props: WooCommerceInstallProps ): ReactElement | null {
+	const { goToStep } = props;
+	const { __ } = useI18n();
+
+	// todo: we should allow any (non jetpack, non p2) site to show up here,
+	// and push people through the checkout step as needed
```

Comment:
what's the plan for this? Could we just simply add a `get a Business plan` button for now?

Reply:
That's a good idea.

Reply:
<img width="1058" alt="Screen Shot 2021-11-25 at 2 14 02 AM" src="https://user-images.githubusercontent.com/77539/143383594-745b79bb-9315-428f-92d2-afa51404e7bc.png">

something like :this:

Reply:
Changed this to pass through any sites with free/personal/prem/biz/ecom plans. (don't want p2 showing up here or jetpack for now.)